code/game/machinery/hologram.dm

```diff
@@ -217,6 +217,12 @@ Possible to do for anyone motivated enough:
 	default_unfasten_wrench(user, tool)
 	return TOOL_ACT_TOOLTYPE_SUCCESS
 
+/obj/machinery/holopad/set_anchored(anchorvalue)
+	. = ..()
+	if(isnull(.) || !outgoing_call || anchorvalue)
```

Comment:
Can you call while unanchored? If so, shouldn't we also cancel the call if wrenched back in?

Reply:
No, it's not functional when it's not anchored.

Reply:
It seems we've got #68898 in a similar vein to the issue this PR fixed, is this related to the point brought up in this comment?